src/views/menus/useGlobalCommands.tsx

```diff
@@ -148,21 +144,15 @@ export const useGlobalCommands = (): MenuConfig<string, string> => {
     //   ],
     // },
     {
-      group: 'navigation',
-      groupLabel: 'Navigation',
-      items: [
-        {
-          value: NavItems.NavigateToMarket,
-          label: 'Navigate to Market',
-          subitems: joinedPerpetualMarketsAndAssets.map(({ market, name, id }) => ({
-            value: market ?? '',
-            slotBefore: <AssetIcon symbol={id} />,
-            label: name ?? '',
-            tag: id,
-            onSelect: () => navigate(`${AppRoute.Trade}/${market}`),
-          })),
-        },
-      ],
+      group: 'markets',
+      groupLabel: 'Markets',
```

Comment:
tiny nit, which might be irrelevant given that idk how many external users know this cmd+k shortcut exists, but we could probably internationalize this with `stringGetter` and existing [keys](https://github.com/dydxprotocol/v4-localization/blob/main/config/localization/en/app.json#L217)